runtime/tr.source/trj9/codegen/J9CodeGenerator.cpp

```diff
@@ -2388,6 +2388,13 @@ J9::CodeGenerator::populateOSRBuffer()
             }
          }
 
+      /*
+       * dead slots are bookkept together with shared slots under voluntary OSR
```

Comment:
Shouldn't this specify involuntary OSR, not voluntary?

Reply:
Thanks for pointing this out.